include/deal.II/distributed/shared_tria.h

```diff
@@ -111,6 +111,25 @@ namespace parallel
                                          const SubCellData &subcelldata);
 
       /**
+       * Copy @p old_tria to this triangulation.
+       *
+       * This function also partitions triangulation based on the MPI
+       * communicator provided to constructor.
+       */
+      virtual void copy_triangulation (const dealii::Triangulation<dim, spacedim> &old_tria);
```

Comment:
``other_tria`` is probably a better name than ``old_tria``? Can you document for which implementation this will work (serial, p::d, p::s, ...)?